cmake/edge_configure.cmake

```diff
@@ -109,9 +109,23 @@ if (${FIRMWARE_UPDATE})
   MESSAGE ("Enabling firmware update for Edge")
 
   if (FOTA_ENABLE)
-    execute_process ( COMMAND cp ${CMAKE_CURRENT_SOURCE_DIR}/fota/fota_install_callback.c ${CMAKE_CURRENT_SOURCE_DIR}/lib/mbed-cloud-client/fota/ )
+    execute_process ( COMMAND cp ${CMAKE_CURRENT_SOURCE_DIR}/fota/fota_app_callbacks.c ${CMAKE_CURRENT_SOURCE_DIR}/lib/mbed-cloud-client/fota/ )
+    if (NOT DEFINED FOTA_SCRIPT_DIR)
+      SET (FOTA_SCRIPT_DIR \"/opt/pelion\")
+    endif()
+    if (NOT DEFINED FOTA_INSTALL_MAIN_SCRIPT)
+      SET (FOTA_SCRIPT_DIR \"fota_update_activate.sh\")
```

Comment:
```suggestion
      SET (FOTA_INSTALL_MAIN_SCRIPT \"fota_update_activate.sh\")
```

Reply:
Will fix, but why not `FOTA_INSTALL_MAIN_SCRIPT_DIR`?